packages/component-library/package.json

```diff
@@ -35,7 +35,7 @@
     "@kaizen/hosted-assets": "^1.1.0",
     "@types/classnames": "^2.2.6",
     "@types/lodash": "^4.14.132",
-    "@types/uuid": "^8.3.0",
+    "@types/uuid": "^3.3.2",
```

Comment:
Is it intentional to go back major versions?

Reply:
It seems this now matches the actual library version we're using (the types package had run ahead of the actual package)

Reply:
Yeah exactly I think it's meant to be consistent. I had to do this because there was a type error when importing from it.